themes/bootstrap3/templates/RecordDriver/DefaultRecord/explain-line.phtml

```diff
@@ -0,0 +1,36 @@
+<tr>
+  <td class="percentage">
+    <?=$this->localizedNumber($explainElement['percent'], $decimalPlaces)?>%
+  </td>
+  <td>
+    <?php
+    if (count($fieldName) > 1) {
+      echo $this->transEsc('Synonym') . ': [';
+    }
+
+    for ($i = 0; $i < count($fieldName); $i++) {
+      $name = $fieldName[$i];
+      $value = $fieldValue[$i] ?? '';
+      $exactMatch = $explainElement['exactMatch'][$i] ?? '';
+      if (!empty($name)) {
+        $description = $this->translate('IndexFieldDescription::' . $name);
+        $description = $name !== $description ? $description : '';
+      } else {
+        $description = '';
+      }
+      $exactMatchClass = ($exactMatch ?? 'unkown') . '-match';
```

Comment:
Typo: unkown (and if this is really going to be displayed, should it be translated?)

Reply:
fixed. This is a class and is not displayed directly. So it does not need to be translated.

Reply:
Of course! Sorry for misreading the context. :-)